lawnchair/src/app/lawnchair/ui/preferences/components/WarningPreference.kt

```diff
@@ -0,0 +1,33 @@
+package app.lawnchair.ui.preferences.components
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.Warning
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+
+@Composable
+fun WarningPreference(
+    modifier: Modifier = Modifier,
+    text: String,
+) {
+    PreferenceTemplate(
+        modifier = modifier,
+        title = {},
+        description = {
+            Text(
+                text = text,
+                color = MaterialTheme.colorScheme.error,
+            )
+        },
+        startWidget = {
+            Icon(
+                imageVector = Icons.Outlined.Warning,
```

Comment:
Please change it to use the rounded icons to be consistent with other part of the UI.

Reply:
My bad, fixed.